tests/unit/layout/size_of.rs

```diff
@@ -5,5 +5,5 @@
 use layout::Fragment;
 use layout::SpecificFragmentInfo;
 
-size_of_test!(test_size_of_fragment, Fragment, 160);
+size_of_test!(test_size_of_fragment, Fragment, 136);
 size_of_test!(test_size_of_specific_fragment_info, SpecificFragmentInfo, 24);
```

Comment:
Because locally it says:

```
---- size_of::test_size_of_fragment stdout ----
	thread 'size_of::test_size_of_fragment' panicked at 'Your changes have decreased the stack size of Fragment from 160 to 136. Good work! Please update the expected size in /Users/nox/src/servo/tests/unit/layout/size_of.rs.', /Users/nox/src/servo/tests/unit/layout/size_of.rs:8
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
stack backtrace:
   0: std::sys::imp::backtrace::tracing::imp::unwind_backtrace
   1: std::panicking::default_hook::{{closure}}
   2: std::panicking::default_hook
   3: std::panicking::rust_panic_with_hook
   4: std::panicking::begin_panic
   5: std::panicking::begin_panic_fmt
   6: layout_tests::size_of::test_size_of_fragment
   7: <F as test::FnBox<T>>::call_box
   8: __rust_maybe_catch_panic
```